conda/build.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh
+
+NPROC=`nproc`
+
+git checkout tags/v0.87.5
+
+mkdir build
+
+cd build
+
+cmake \
+-DCMAKE_INSTALL_PREFIX="${PREFIX}" \
+-DCMAKE_CXX_COMPILER="${PREFIX}/bin/g++" \
+-DCMAKE_C_COMPILER="${PREFIX}/bin/gcc" \
+-DCMAKE_Fortran_COMPILER="${PREFIX}/bin/gfortran" \
+-DEL_USE_64BIT_INTS=ON \
+-DEL_HAVE_QUADMATH=OFF \
+-DCMAKE_BUILD_TYPE=Release \
+-DEL_HYBRID=ON \
```

Comment:
I would recommend not enabling the Hybrid build by default: is there a good reason to do so?